api/src/test/java/com/xebia/starters/api/ApiArchitectureTest.java

```diff
@@ -0,0 +1,22 @@
+package com.xebia.starters.api;
+
+import com.github.xebia.archunit.AbstractArchitectureTests;
+import com.tngtech.archunit.core.importer.ClassFileImporter;
+import com.tngtech.archunit.core.importer.ImportOption;
+import com.xebia.starters.Application;
+import com.xebia.starters.domain.Entity;
+
+class ApiArchitectureTest extends AbstractArchitectureTests {
+
+
+    public ApiArchitectureTest() {
+        super(new ClassFileImporter()
+                        .withImportOption(ImportOption.Predefined.DO_NOT_INCLUDE_TESTS)
+                        .importPackagesOf(Application.class),
+                new String[]{
+                        Entity.class.getPackage().getName()
```

Comment:
I feel like this way of passing in entity class package names is a bit awkward. I had to create a dummy class just to pass failing test. Should we tweak this in the archunit-extension somehow?

Reply:
I think we can make it optional. We can ignore the tests that don't have this set.